tests/application/assets/Room.ts

```diff
@@ -0,0 +1,18 @@
+import { AssetModelDefinition } from "../../../index";
+
+export const roomAssetDefinition: AssetModelDefinition = {
+  measures: [
+    {
+      name: "temperature",
+      type: "temperature",
+    },
+    {
+      name: "humidity",
+      type: "humidity",
+    },
+  ],
```

Comment:
You could define the same measures as in the Room asset model from [iot-platform](https://github.com/kuzzleio/iot-platform/blob/3-dev/apps/api/lib/modules/air_quality/assets/Room.ts)
```suggestion
  measures: [
    {
      name: "temperature",
      type: "temperature",
    },
    {
      name: "humidity",
      type: "humidity",
    },
    {
      name: "co2",
      type: "co2",
    },
    {
      name: "illuminance",
      type: "illuminance",
    },
  ],
```